packages/mdc-drawer/modal/mdc-drawer-modal.scss

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/animation/functions";
+@import "@material/elevation/mixins";
+@import "@material/rtl/mixins";
+@import "../mixins";
+@import "../variables";
+
+.mdc-drawer--modal {
+  @include mdc-drawer-scrim-fill-color($mdc-drawer-modal-scrim-color, $mdc-drawer-modal-scrim-opacity);
+  @include mdc-elevation($mdc-drawer-modal-elevation);
+  @include mdc-rtl-reflexive-position(left, 0);
+
+  display: none;
+  position: fixed;
+
+  &.mdc-drawer--open {
+    display: block;
+  }
+}
+
+.mdc-drawer-scrim {
+  display: none;
+  position: fixed;
+  top: 0;
+  right: 0;
```

Comment:
I think you can omit right/bottom since you have width/height? (At least that's what I've seen in our existing code for dialog and drawer)